packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/openshiftSecretFieldValidator.ts

```diff
@@ -41,7 +44,7 @@ export const openshiftSecretFieldValidator = (id: string, value: string): Valida
 const validateToken = (value: string): ValidationMsg => {
   const valid = validateK8sToken(value);
 
-  if (value === '') {
+  if (value === undefined || value === '') {
```

Comment:
the idea is differentiate between `undefined` that will return `default` , for a newly opened form with no user interaction, and `""` empty string for missing value `error` , checking `(value === undefined || value === '') ` does not look right, what am i missing ?

Reply:
Correct and that's how it is implemented for other providers.
Specifically for the OCP provider in which the `token` field is not mandatory, I didn't see any advantage for differentiating those 2 states of empty and initialized, i.e. if the field is left empty then it's left grey since marking that as valid/green will be problematic  in case the `url` field is not set. I can differentiate if you think it's better. 

BTW, the same handling goes for the `cacert` field as well.